saleor/checkout/migrations/0058_auto_20230313_1602.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.18 on 2023-03-13 16:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("checkout", "0057_remove_metadata_from_checkout_model_state"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="checkout",
+            name="authorize_status",
+            field=models.CharField(
+                choices=[
+                    ("none", "The checkout is not authorized"),
+                    ("partial", "The checkout is partially authorized"),
+                    ("full", "The checkout is fully authorized"),
+                ],
+                db_index=True,
+                default="none",
```

Comment:
This will set `none` value only on existing rows, we should also add the SQL operation for setting the default `none` value on this column

Reply:
- checkout on main
- populatedb on main
- checkout to this branch
- ./manage.py migrate
- and psql returns data as below:
![image](https://user-images.githubusercontent.com/23282005/225581711-b6e697a9-824d-4c69-a4f3-f403edfc86eb.png)

So it looks like, the status is populated for all existing fields. 
Also we will handle a zero downtime migrations in separate PR so we will double check this.

cc @kadewu